examples/model-drivers/LennardJones612__MD_414112407348_003/LennardJones612Implementation.cpp

```diff
@@ -299,16 +299,19 @@ int LennardJones612Implementation::OpenParameterFiles(
     return ier;
   }
 
+  std::string const * paramFileDirName;
+  modelDriverCreate->GetParameterFileDirectoryName(&paramFileDirName);
   for (int i = 0; i < numberParameterFiles; ++i)
   {
     std::string const * paramFileName;
-    ier = modelDriverCreate->GetParameterFileName(i, &paramFileName);
```

Comment:
@ellio167 I think we should keep this functionality and have both ways to get the parameter files.